tests/tests/test-version.ts

```diff
@@ -9,3 +10,12 @@ describeDevMoonbeam("Version RPC", (context) => {
     expect(await context.web3.eth.net.getId()).to.equal(1281);
   });
 });
+
+describeDevMoonbeam("Version - ChainId", (context) => {
+  it("should be accessible within a contract", async function () {
+    const { contract, rawTx } = await createContract(context.web3, "CheckBlockGasLimit");
```

Comment:
why is a contract deployed here?

Reply:
I should have split into a different contract (or rename it) but the idea is to see if the chainId is accessible through a smart contract